crates/ruff_python_formatter/src/statement/stmt_class_def.rs

```diff
@@ -33,6 +34,28 @@ impl FormatNodeRule<StmtClassDef> for FormatStmtClassDef {
 
         write!(f, [text("class"), space(), name.format()])?;
 
+        if !type_params.is_empty() {
+            let sequence_end = {
```

Comment:
Nit: Use `first()` to avoid `unwrap()` and `if` can be used in expression positions.


```suggestion
            let sequence_end = if let Some(first) = bases.first() {
				first.start()
			} else if Some(keyword) = keywords.first() {
				keyword.start()
			} else {
				body.first().unwrap().start()
			};
```